lib/archive.ts

```diff
@@ -73,7 +73,7 @@ async function copySourceToDest(
   { sourceDir, includesRootDir = true }: CopySourceToDestOptions = {}
 ): Promise<boolean> {
   try {
-    logger.log(i18n(`${i18nKey}.copySourceToDest.init`));
+    logger.debug(i18n(`${i18nKey}.copySourceToDest.init`));
```

Comment:
Why were these changed to debug logs?

Reply:
The logs were appearing unexpectedly when I was downloading the files for the new post-migration project. I was thinking of adding a boolean to switch between debug/log but wasn't sure--meant to leave a note with a question but then John was angsty to get the changes out and I forgot 😬

Reply:
Discussed this with @camden11 in a Huddle, and he suggested we add a `hideLogs` boolean, so we don't cause any issues with other commands that use these functions.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/130/commits/d86aa5e928b94628db2a49eca86371ba33e2db70